### ISSUE TITLE
Focus loss with disabled form controls

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1157]
## Context

In #1124, we discussed changing `disabled` attributes / properties to `isDisabled`.

After giving it some thoughts, we propose the following:

- introduce a new `waiting` prop / attribute for all form controls,
- this prop / attribute should be used when you want to disable temporarily a form control and preserve focus (for instance: during an API call),
- if you need to actually disable an element (more permanently), you can still use the `disabled` prop / attribute.

To achieve this, the `waiting` prop / attribute should trigger the following behavior inside the component:

- add the `aria-disabled` attribute to the native element,
- prevent all actions (no implicit submit, no keyboard, click, change events).

## Impacted components:

- [ ] `cc-input-text`,
- [ ] `cc-input-date`,
- [ ] `cc-input-number`,
- [ ] `cc-select`.

**Comment by florian-sanders-cc** [comment]
Closing this one because it's a duplicate of #1102.
We decided to rely on `readonly` instead of introducing a new prop on form controls (see #1102 for more info).
